Bychynski_Pavel_HW_9_2.py

```diff
@@ -0,0 +1,32 @@
+# pizza time
+''' Unique error for pizza issues instead any exceptions '''
+class PizzaError(Exception):
+    def __init__(self, pizza, message):
+        Exception.__init__(self, message)
+        self.pizza = pizza
+
+''' Subclass for cheese errors '''
+class TooMuchCheeseError(PizzaError):
+    def __init__(self, pizza, cheese, message):
+        PizzaError.__init__(self, pizza, message)
+        self.cheese = cheese
+
+''' Main class for generating pizzas from list '''
+class Pizza():
+    def make_pizza(self, pizza, cheese):
+        if pizza not in ["margherita", "capricciosa", "calzone"]:
+            raise PizzaError (pizza, "pizza not in menu")
+        if cheese > 100:
+            raise TooMuchCheeseError (pizza, cheese, "to much cheese")
+        print("Pizza ready!")
```

Comment:
Where is the __init__() method for Pizza class?
It is really important

Try filling out the following statement::
```suggestion
class Pizza():
    def _....__(self):
        self.__pzz_list = ["................................"]
    def make_pizza(self, pizza, cheese):
        self.pizza = pizza
        self.cheese = ....................
        if self.pizza not in self.__pz_list:
            raise PizzaError (self.pizza, "pizza not in menu")
        if .......cheese > 100:
            raise TooMuchCheeseError (self.pizza, self.ch.........., "to much cheese")
        print("Pizza ready!")
```